src/main/java/org/maxgamer/QuickShop/Util/Util.java

```diff
@@ -588,20 +590,13 @@ public static boolean isBlacklisted(Material m) {
      * @return The block the sign is attached to
      */
     public static Block getAttached(Block b) {
-        try {
-            final Sign sign = (Sign) b.getState().getData(); // Throws a NPE
-                                                             // sometimes??
-            final BlockFace attached = sign.getAttachedFace();
-
-            if (attached == null) {
-                return null;
-            }
-            return b.getRelative(attached);
-        } catch (final NullPointerException e) {
-            return null; // /Not sure what causes this.
+        if (b.getBlockData() instanceof Directional) {
+            Directional directional = (Directional) b.getBlockData();
+            return b.getRelative(directional.getFacing().getOppositeFace());
+        } else {
+            return null;
         }
     }
-
```

Comment:
Just because I'm pedantic, can you add this blank line back? :stuck_out_tongue_closed_eyes: